tools/generator/files/api.hh.jinja2

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2012-2020 Association Prologin <association@prologin.org>
+
+#pragma once
+
+#include <memory>
+{#- The following two headers may not be necessary. They are only used if there
+is a std::string, or a std::vector in the generated function signatures.
+We could make the test here and add the includes only if necessary, but it does
+not cost must to include them in every cases. #}
```

Comment:
I think you can remove this comment, because it's probably going to show up in a lot of files, and it doesn't really matter as you say.

Reply:
ok, I added this in the very beginning but I’ve been less verbose since